bitnami/mongodb/Chart.yaml

```
@@ -7,7 +7,7 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     tags:
       - bitnami-common
-    version: 2.x.x
+    version: 2.2.x
```

We usually use the major version here
```suggestion
    version: 2.x.x
```